tests/testproject/TestProject.IntegrationServiceA/Garnet/GarnetExtensions.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using StackExchange.Redis;
+
+public static class GarnetExtensions
+{
+    public static void MapGarnetApi(this WebApplication app)
+    {
+        app.MapGet("/garnet/verify", VerifyGarnetAsync);
```

Comment:
Need to also add a test for this endpoint here:

https://github.com/dotnet/aspire/blob/8836eb4d10c7b6807c6456f0d8b4a606c6b84ea8/tests/Aspire.EndToEnd.Tests/IntegrationServicesTests.cs#L26-L34